shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -968,12 +960,12 @@ poolRewards
   -> Coin
   -> Coin
 poolRewards d_ sigma blocksN blocksTotal (Coin maxP) =
-  if intervalValue d_ < 0.8
-    then floor (p * fromIntegral maxP)
-    else 1
+  floor (pBar * fromIntegral maxP)
   where
-    p = beta / intervalValue sigma
     beta = fromIntegral blocksN / fromIntegral (max 1 blocksTotal)
+    pBar = if intervalValue d_ < 0.8
+             then beta / intervalValue sigma
```

Comment:
Can `intervalValue sigma` be zero?

Reply:
good catch, indeed it's possible. I caught this in the pool ranking module, but missed it here.

Reply:
I only noticed it here because I saw you catching it there 😉